modules/rdma-creating-a-service-user.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * hardware_accelerators/rdma-remote-direct-memory-access.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rdma-creating-a-service-user_{context}"]
+
+= Creating a service user
+
```

Comment:
This module also needs a short intro/abstract

Reply:
Added.